_im-plugin/index-transforms/transforms-apis.md

```diff
@@ -595,6 +598,12 @@ GET _plugins/_transform/<transform_id>/_explain
   "sample": {
     "metadata_id": "PzmjweME5xbgkenl9UpsYw",
     "transform_metadata": {
+      "continuous_stats": {
+      "last_timestamp": 1621883525672,
+      "documents_behind": {
+        "sample_index": 72
+        }
+      },
```

Comment:
Not sure if this formatting is preserved in the actual website, but if it is, could you update the spacing to:
```
"continuous_stats": {
    "last_timestamp": 1621883525672,
    "documents_behind": {
        "sample_index": 72
    }
},
```
This will make it easier for users to understand the json formatting.

Reply:
Another oops from me. Fixed indenting.